jukebot/config/info.github.py

```diff
@@ -0,0 +1 @@
+TOKEN = '<DEFINED IN token generation in app page>'
```

Comment:
i have a file named info.py with the generated token. yes u could also set an environment variable but i just put info.py into gitignore